src/hooks/use-comments.ts

```diff
@@ -1,322 +0,0 @@
-/*!
- * The contents of this file are subject to the Common Public Attribution License Version 1.0 (the “License”); you may not use this file except in compliance with the License. You may obtain a copy of the License at https://mpapp-public.gitlab.io/manuscripts-frontend/LICENSE. The License is based on the Mozilla Public License Version 1.1 but Sections 14 and 15 have been added to cover use of software over a computer network and provide for limited attribution for the Original Developer. In addition, Exhibit A has been modified to be consistent with Exhibit B.
- *
- * Software distributed under the License is distributed on an “AS IS” basis, WITHOUT WARRANTY OF ANY KIND, either express or implied. See the License for the specific language governing rights and limitations under the License.
- *
- * The Original Code is manuscripts-frontend.
- *
- * The Original Developer is the Initial Developer. The Initial Developer of the Original Code is Atypon Systems LLC.
- *
- * All portions of the code written by Atypon Systems LLC are Copyright (c) 2019 Atypon Systems LLC. All Rights Reserved.
- */
-
-import {
-  Build,
-  buildComment,
-  buildContribution,
-  findNodePositions,
-  ManuscriptEditorState,
-  ManuscriptSchema,
-  UserProfileWithAvatar,
-} from '@manuscripts/manuscript-transform'
-import {
-  CommentAnnotation,
-  Contribution,
-} from '@manuscripts/manuscripts-json-schema'
-import {
-  Annotation,
-  commands,
-  focusedEntities,
-  getTrackPluginState,
-} from '@manuscripts/track-changes'
-import { Command } from 'prosemirror-commands'
-import { Node as ProsemirrorNode } from 'prosemirror-model'
-import { useCallback, useEffect, useState } from 'react'
-
-import { CommentFilter } from '../components/projects/CommentListPatterns'
-import { useStore } from '../store'
-import { useMicrostore } from './use-microstore'
-
-interface UnsavedComment extends Build<CommentAnnotation> {
-  contributions: Contribution[]
-}
-
-type CommentState = Array<{
-  comment: CommentAnnotation | UnsavedComment
-  annotation?: Annotation
-  saveStatus: string
-}>
-
-export const getInitialCommentState = (
-  comments: CommentAnnotation[],
-  annotations: Annotation[]
-): CommentState => {
-  return comments.map((comment) => ({
-    comment,
-    annotation: annotations.find(
-      (annotation) => annotation.uid === comment.target
-    ),
-    saveStatus: '',
-  }))
-}
-
-export const updateComment = (
```

Comment:
You will need to update the tests ( `/src/hooks/__tests__/use-comments.test.ts`.)